src/locale/locales/fr/messages.po

```diff
@@ -963,12 +948,12 @@ msgstr "Ignorer le brouillon"
 
 #: src/view/screens/Moderation.tsx:207
 msgid "Discourage apps from showing my account to logged-out users"
-msgstr "Empêcher les applis de montrer mon compte aux utilisateurs déconnectés"
+msgstr "Empêcher les applis de montrer mon compte aux personnes déconnectées"
```

Comment:
"personnes déconnectées" => "personnes non connectées"
"non connectées" implies more clearly "people that never connected/don't have an account", I think